pkg/kubectl/resource/visitor.go

```diff
@@ -154,6 +154,17 @@ func (i *Info) Refresh(obj runtime.Object, ignoreError bool) error {
 	return nil
 }
 
+// String returns the general purpose string representation
+func (i *Info) String() string {
+	basicInfo := fmt.Sprintf("Name: %q, Namespace: %q\nObject: %+q", i.Name, i.Namespace, i.Object)
```

Comment:
> Object: %+q

Can you paste a example output here?
Do you think using `%+q` will be better than `%#v`?

Reply:
Here is the example output of `%+q`
```shell

F0222 12:18:31.699709   77239 helpers.go:119] error: error when retrieving current configuration of:
Resource: "pods", Version: "/v1, Kind=Pod"
Name: "", Namespace: "default"
Object: &{map["apiVersion":"v1" "kind":"Pod" "metadata":map["generateName":"testing-nnp-" "namespace":"default" "annotations":map["kubectl.kubernetes.io/last-applied-configuration":""]] "spec":map["restartPolicy":"Never" "containers":[map["command":["grep NoNewPriv /proc/self/status"] "image":"busybox" "name":"simple-container"]] "initContainers":[map["command":["grep NoNewPriv /proc/self/status"] "image":"busybox" "name":"init-container"]]]]}
from server for: "lw.yml": resource name may not be empty
```
And output of `%#v`
```shell

F0222 12:21:28.568327   81874 helpers.go:119] error: error when retrieving current configuration of:
Resource: "pods", Version: "/v1, Kind=Pod"
Name: "", Namespace: "default"
Object: &unstructured.Unstructured{Object:map[string]interface {}{"apiVersion":"v1", "kind":"Pod", "metadata":map[string]interface {}{"generateName":"testing-nnp-", "namespace":"default", "annotations":map[string]interface {}{"kubectl.kubernetes.io/last-applied-configuration":""}}, "spec":map[string]interface {}{"containers":[]interface {}{map[string]interface {}{"command":[]interface {}{"grep NoNewPriv /proc/self/status"}, "image":"busybox", "name":"simple-container"}}, "initContainers":[]interface {}{map[string]interface {}{"command":[]interface {}{"grep NoNewPriv /proc/self/status"}, "image":"busybox", "name":"init-container"}}, "restartPolicy":"Never"}}}
```

Reply:
>
```
+	always print a sign for numeric values;
	guarantee ASCII-only output for %q (%+q)
```

I think `%+q` is fine.

Reply:
：)